docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md

```diff
@@ -15,14 +15,14 @@ This statement pauses a Routine Load job but does not terminate this job. You ca
 ## Syntax
 
 ```SQL
-PAUSE ROUTINE LOAD FOR <db_name>.<job_name>;
+PAUSE ROUTINE LOAD FOR [db_name.]<job_name>;
 ```
 
 ## Parameters
 
 | Parameter | Required | Description                                                  |
 | --------- | -------- | ------------------------------------------------------------ |
-| db_name   |          | The name of the database for which you want to pause a Routine Load job. |
+| db_name   |          | The name of the database where the Routine Load job belongs. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/1dacaaa4-54d3-40c3-a8fe-64ff666df469)
-> to which